deploy/template.yaml

```diff
@@ -359,6 +359,34 @@ Resources:
         Owner: !Ref OwnerTagValue
         Source: !Ref SourceTagValue
 
+  APIGatewayIDParameter:
```

Comment:
Could we move the `APIGatewayIDParameter` resource to the API Gateway section please

Reply:
Thanks Dom all edited now 👍